README.md

```diff
@@ -33,4 +33,15 @@ This project is perfect for people starting out with opensource and knows basics
 <img src="https://github.com/mudit023/templates/blob/main/theme-toggle/Screenshot-Mobile.png" width=400px height=518px align="left">
 <img src="https://github.com/mudit023/templates/blob/main/background-slider/Screenshot-Mobile.png" width=400px>
 
-### Want to contribute to the project? Read the [Contribution Guidelines](https://github.com/mudit023/templates/blob/main/CONTRIBUTING.md), [Template Guidelines](https://github.com/mudit023/templates/blob/main/TEMPLATE_GUIDELINES.md), and adhere to our [Code of Conduct](https://github.com/mudit023/templates/blob/main/CODE_OF_CONDUCT.md).
+**Note: The first thing you can contribute to is this README.**
```

Comment:
Still you are undoing changes i have made in the readme.

Reply:
Ok, Please Push these changes to update

Reply:
If i will merge this PR the changes I have made in the README will get reverted.